### PR TITLE
#issues #1 #2 email handling

### LINKED ISSUE #1: Search for On Boarding Email
* Connect to your Outlook mail account
* Search through all unread/read emails in inbox folder for an email with a specific subject "Weekly On Boarding Candidate List"
* Check if email is present or not
* If emails not present end the process
* If email's present open it
* Check if open email has an excel attachment
* If attachment present download the attachment
* If not present get the senders email address & throw Business Exception #4 

Don't forget to use annotations on major sequences plus renaming your activities, variables & objectives following best practices

### LINKED ISSUE #4: Implement Error Handling for Missing Excel Attachment
If excel attachment isn't present the following happen #1   :-

* Handle System Exception & send email in case an error occurs during runtime
* Handle Business Exception & send an email to the Sender email address informing them that the email was received but doesn't have the excel attachment
* Move the email to a different folder #6  and end the process

Don't forget to use annotations on major sequences plus renaming your activities, variables & objectives following best practices

### PR DESCRIPTION
**Pull request by mbunix**
Added Email handling Functionality and Error  Handler

### CODE REVIEW

File: Framework/InitAllApplications.xaml
Comment (FreddieMwau) [inline Framework/InitAllApplications.xaml:1]: There are a few changes needed:
* Wrap all activities with a Try Catch block
* Store system & business exception emails in the Constants Tab of the Config file
* Avoid hard coding values by using variables & arguments to hold them
* Follow best practices for naming variables, i,e "str_Outlook_Account"

**Review by FreddieMwau — CHANGES_REQUESTED** [review]
Great work on the solution 👍🏽 ,  just make the changes and we're good

File: Framework/InitAllApplications.xaml
Comment (mbunix) [reply Framework/InitAllApplications.xaml:1]: Fixed changes requested

Comment (FreddieMwau) [reply Framework/InitAllApplications.xaml:1]: Push the changes & I'll review them

**Review by FreddieMwau — COMMENTED** [review]
Changes implemented in a different branch

**Pull request closed without merging**